Assets/MixedRealityToolkit/Inspectors/MixedRealityToolkitInspector.cs

```diff
@@ -118,11 +143,267 @@ public override void OnInspectorGUI()
+                SearchFieldString = SessionState.GetString(searchDisplaySearchFieldKey, string.Empty),
+                RequireAllSubjects = SessionState.GetBool(searchDisplayRequireAllTagsKey, true),
+                RequireAllKeywords = SessionState.GetBool(searchDisplayRequireAllKeywordsKey, true),
+                SearchTooltips = SessionState.GetBool(searchDisplaySearchTooltipsKey, true),
+                SearchFieldObjectNames = SessionState.GetBool(searchDisplaySearchFieldObjectNamesKey, true),
+                SearchChildProperties = SessionState.GetBool(searchDisplaySearchChildPropertiesKey, true)
+            };
+
+            // Draw our subject tags in a clump
+            EditorGUILayout.HelpBox("Select subjects below to search for profiles tagged with that subject.", MessageType.Info);
+
+            #region draw subjects and field input
+
+            using (new EditorGUILayout.VerticalScope())
+            {
+                bool horizontalGroupOpen = false;
+                int numButtons = maxSubjectButtonsPerLine;
+
+                using (new EditorGUILayout.HorizontalScope())
+                {
+                    GUI.color = config.SelectedSubjects == 0 ? MixedRealityInspectorUtility.SuccessColor : MixedRealityInspectorUtility.DisabledColor;
+                    if (GUILayout.Button("(None)", EditorStyles.miniButton))
+                    {
+                        config.SelectedSubjects = 0;
+                    }
+
+                    GUI.color = config.SelectedSubjects == SubjectTag.All ? MixedRealityInspectorUtility.SuccessColor : MixedRealityInspectorUtility.DisabledColor;
+                    if (GUILayout.Button("(All)", EditorStyles.miniButton))
+                    {
+                        config.SelectedSubjects = SubjectTag.All;
+                    }
+                }
+
+                List<SubjectTag> sortedTags = new List<SubjectTag>();
+                foreach (SubjectTag tag in Enum.GetValues(typeof(SubjectTag)))
+                {
+                    if (tag == SubjectTag.All)
+                    {
+                        continue;
+                    }
+
+                    sortedTags.Add(tag);
+                }
+
+                sortedTags.Sort(delegate (SubjectTag t1, SubjectTag t2) { return t1.ToString().CompareTo(t2.ToString()); });
+
+                foreach (SubjectTag tag in sortedTags)
+                {
+                    if (numButtons >= maxSubjectButtonsPerLine)
+                    {
+                        numButtons = 0;
+                        EditorGUILayout.BeginHorizontal();
+                        horizontalGroupOpen = true;
+                    }
+
+
+                    bool selected = (config.SelectedSubjects & tag) != 0;
+                    GUI.color = selected ? MixedRealityInspectorUtility.SuccessColor : MixedRealityInspectorUtility.DisabledColor;
+                    if (GUILayout.Button(tag.ToString(), EditorStyles.miniButton))
+                    {
```

Comment:
Obviously remove based on comments in main PR section. Nice added benefit is this will simplify the code up a lot too